etc/generate-envoy-config/envoy.libsonnet

```diff
@@ -70,7 +70,7 @@
     },
   },
 
-  loadAssignment(name, address, port): {
+  loadAssignment(name, address, port):: {
```

Comment:
?

Reply:
This makes it non-exported.  See "visibilities" in https://jsonnet.org/ref/language.html